test/unittests/capi_test.cpp

```diff
@@ -98,6 +98,40 @@ TEST(capi, find_exported_function)
     fizzy_free_module(module);
 }
 
+TEST(capi, find_exported_global)
+{
+    /* wat2wasm
+    (module
+      (func $f (export "foo") (result i32) (i32.const 42))
+      (global (export "g1") i32 (i32.const 42))
+      (table (export "tab") 0 anyfunc)
+      (memory (export "mem") 1 2)
+    )
+    */
+    const auto wasm = from_hex(
+        "0061736d010000000105016000017f030201000404017000000504010101020606017f00412a0b07180403666f"
+        "6f00000267310300037461620100036d656d02000a06010400412a0b");
+
+    auto module = fizzy_parse(wasm.data(), wasm.size());
+    ASSERT_NE(module, nullptr);
+
+    auto instance = fizzy_instantiate(module, nullptr, 0, nullptr, 0);
+    ASSERT_NE(instance, nullptr);
+
+    FizzyExternalGlobal global;
+    ASSERT_TRUE(fizzy_find_exported_global(instance, "g1", &global));
+    EXPECT_EQ(global.type.value_type, FizzyValueTypeI32);
+    EXPECT_FALSE(global.type.is_mutable);
+    EXPECT_EQ(global.value->i64, 42);
+
+    EXPECT_FALSE(fizzy_find_exported_global(instance, "g2", &global));
+    EXPECT_FALSE(fizzy_find_exported_global(instance, "foo", &global));
+    EXPECT_FALSE(fizzy_find_exported_global(instance, "tab", &global));
+    EXPECT_FALSE(fizzy_find_exported_global(instance, "mem", &global));
```

Comment:
Why are these failing?

Reply:
first one doesn't exist, and others are not globals.

Reply:
Ah, for a second thought this is called `find_export` and handles all kinds.